src/DataTable/tests/BulkActions.test.jsx

```diff
@@ -82,46 +82,63 @@ describe('<BulkActions />', () => {
   describe('with functional rendering', () => {
     it('renders the function', () => {
       const myFunction = () => <Button>Some Button</Button>;
-      const wrapper = mount(<BulkActionsWrapper value={{ ...instance, bulkActions: myFunction }} />);
-      const button = wrapper.find(Button);
-      expect(button.length).toEqual(1);
+      const { container } = render(<BulkActionsWrapper value={{ ...instance, bulkActions: myFunction }} />);
+      const button = within(container).getByText('Some Button');
+      expect(button).toBeInTheDocument();
     });
   });
+
   describe('with one action', () => {
-    it('displays the primary button as an brand button', () => {
-      const wrapper = mount(<BulkActionsWrapper value={{ ...instance, bulkActions: [<FirstAction />] }} />);
-      const button = wrapper.find(Button);
-      expect(button.length).toEqual(1);
-      expect(button.props().variant).toEqual('brand');
+    it('displays the primary button as a brand button', () => {
+      const { getByTestId } = render(
+        <BulkActionsWrapper
+          value={{ ...instance, bulkActions: [<FirstAction data-testid="brand" />] }}
+        />,
+      );
+      const button = getByTestId('brand');
+      expect(button).toBeInTheDocument();
     });
   });
+
   describe('with two actions', () => {
-    const wrapper = mount(<BulkActionsWrapper value={{ ...instance, bulkActions: twoActions }} />);
-    it('displays the user\'s first button as an brand button', () => {
-      const buttons = wrapper.find(Button);
+    let getAllByRole;
+    let buttons;
+    beforeEach(() => {
+      const { getAllByRole: gabr } = render(
+        <BulkActionsWrapper
+          value={{ ...instance, bulkActions: twoActions }}
+        />,
+      );
+      getAllByRole = gabr;
```

Comment:
same comment as somewhere above, can we just use `screen.getAllByRole` in these tests? And don't do such stuff in `beforeEach`, it just looks unnecessary